sap/tests/test_sap.py

```diff
@@ -453,6 +453,83 @@ def test_collect_databases(aggregator, instance):
         aggregator.all_metrics_asserted()
 
 
+def test_collect_only_hosts_https(aggregator, https_instance):
+    # TODO this is needed because the topology retains data across tests
+    topology.reset()
+
+    host_control_url = "https://localhost:1129/SAPHostControl"
+    with requests_mock.mock() as m:
+        m.get(host_control_url + "/?wsdl", text=_read_test_file("wsdl/HttpsHostControl.wsdl"))
```

Comment:
is the `HttpsHostControl.wsdl` different from `HostControl.wsdl` ?

Reply:
@TizianoPerrucci yes the URL is changed to `https` URL in the file.